### PR TITLE
Bump event-bus image

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Bump event-bus image
- Bump memory limits for event-bus-publish

**Related issue(s)**

See #6299

### CODE REVIEW

**Comment by sayanh** [comment]
After 5+ hours of publishing events, here's what we see: 

Screenshot 2019-11-20 at 18 48 50
Screenshot 2019-11-20 at 18 48 41

File: resources/event-bus/values.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ global:
         cpu: 20m
         memory: 32Mi
       limits:
-        memory: 32Mi
+        memory: 128Mi
```
Comment (antoineco) [inline resources/event-bus/values.yaml]: Still needed?

**Comment by sayanh** [comment]
@nachtmaar @marcobebway @antoineco With decent traffic of events(refer to the screenshot above), the current usage of publish-service is `23.8Mi` and the memory usage is showing a slight upward tendency. Keep in mind that, this is 5+ hours of data. Hence, I feel, `32Mi` is way too less a limit for this container considering the current usage.

This is definitely an improvement from before where the pod was OOM-ing in 10 minutes.

Now, the primary rationale behind 128Mi is preparing for the worst. Of course, it could have been 64Mi or less. As this will be a pre-mature optimization, a larger number won't hurt IMO.

**Comment by sayanh** [comment]
@nachtmaar The other limits are for subscriptioncontroller as rest are unused. Subscriptioncontroller is unimpacted with this exercise.

**Pull request merged**